tweet-wall-example/script.js

```diff
@@ -1,131 +1,129 @@
-    console.log(data);
-    return data;
+
+    return deckURL;
 }
 
-async function renderTweet(tweet) {
-    const tweetEl = document.createElement('div');
-    tweetEl.classList.add('post-it');
+const deckURL = createDeck();
+console.log(deckURL)
+
+async function reDraw() {
+    const newResponse = await fetch(deckURL); //creates call
+    console.log(deckURL)
+    const newData = await newResponse.json();
+    console.log(newData);
+
+    const cardArray = newData.cards; //short name
+        for (let i = 0; i < cardArray.length; i++){ //goes through every card in the array
+        const idEl = document.createElement('div');
+        contentElement.appendChild(idEl);
+        const newCard = new Card(idEl, cardArray[i].value, cardArray[i].suit, cardArray[i].image)
+        newCard.displayID();
+    }
 
-    const tweetContentEl = document.createElement('p');
-    tweetContentEl.innerHTML = tweet.content;
-    tweetEl.appendChild(tweetContentEl);
 
-    const user = await getUser(tweet.userId);
-    const tweetAuthorEl = document.createElement('p');
-    tweetAuthorEl.innerHTML = `@${user.username}`;
-    tweetEl.appendChild(tweetAuthorEl);
 
-    const deleteButton = document.createElement('button');
-    deleteButton.innerHTML = 'Delete';
-    deleteButton.addEventListener('click', async () => {
-        await deleteTweet(tweet.id);
-        const tweets = await getTweets();
-        displayTweets(tweets);
-    });
-    tweetEl.appendChild(deleteButton);
+    // for (let i = 0; i < cardArray.length; i++){
+    //     const idEl = document.createElement('div');
+    //     contentElement.appendChild(idEl);
+    //     const newCard = new Card(idEl, cardArray[i].value, cardArray[i].suit, cardArray[i].image)
+    //     newCard.displayID();
+    // }
 
-    tweetEl.style.left = randomInt(0, window.innerWidth - 200) + 'px';
-    tweetEl.style.top = randomInt(0, window.innerHeight - 200) + 'px';
+    // if(newData.remaining == 0){
+    //     const resetDeck = await fetch("https://deckofcardsapi.com/api/deck/m08ljrwjejmm/return/"); 
+    //     const resetResponse = await resetDeck.json();
+    //     console.log(resetResponse);
 
-    container.appendChild(tweetEl);
-}
+    //     const reDraw = await fetch("https://deckofcardsapi.com/api/deck/m08ljrwjejmm/draw/?count=1");
+    //     const newResponse = await reDraw.json();
 
-async function displayTweets(tweets) {
-    container.innerHTML = '';
-    tweets.forEach((tweet) => {
-        renderTweet(tweet);
-    });
+    //     for (let i = 0; i < newResponse.length; i++){
+    //         const idEl = document.createElement('div');
+    //         contentElement.appendChild(idEl);
+    //         const newCard = new Card(idEl, cardArray[i].value, cardArray[i].suit, cardArray[i].image)
+    //         newCard.displayID();
+    //     }
+    // }
 }
+const drawbutton = document.querySelector('drawButton')
+const reDrawButton = document.querySelector('reDraw')
```

Comment:
Firstly, your click events are not going to work because you forgot to add `#` when trying to select `drawButton` and `reDraw` elements by ID.